modules/sensor/src/rgb-depth/realsense/vpRealSense2.cpp

```diff
@@ -812,12 +1021,25 @@ void vpRealSense2::getPointcloud(const rs2::depth_frame &depth_frame, const rs2:
 /*!
    Get the extrinsic transformation from one stream to another. This function
    has to be called after open().
-   \param from, to : streams for which the camera extrinsic parameters are returned.
+   \param from, to   : Streams for which the camera extrinsic parameters are returned.
+   \param from_index : Index of the stream from which we will calculate the transformation
   */
-vpHomogeneousMatrix vpRealSense2::getTransformation(const rs2_stream &from, const rs2_stream &to) const
+vpHomogeneousMatrix vpRealSense2::getTransformation(const rs2_stream &from, const rs2_stream &to, int from_index) const
```

Comment:
index description 
completed with: 1: from left to right, 2 from right to left camera for T200 camera models. Otherwise don't use this parameter or set it to -1